src/axom/primal/tests/primal_intersect.cpp

```diff
@@ -1438,11 +1438,10 @@ TEST(primal_intersect, segment_aabb_2d_intersection)
   using BoxType = primal::BoundingBox<double, DIM>;
 
   // Helper lambda for printing out intersection details
-  auto print_details = AXOM_LAMBDA(bool expected,
-                                   const BoxType& b,
-                                   const SegmentType& s,
-                                   const PointType& p)
-  {
+  auto print_details = [=](bool expected,
```

Comment:
To me it looks like the lambdas are identical. Why have two?

Reply:
That's a good point. Does it even have to be a lambda? 

I would have just put that as a free helper function within an anonymous namespace in this file

Reply:
I agree. That would make the code cleaner, IMO. There is really no value in making it a lambda.

Reply:
It it just a short helper function.  

If I moved it to a separate place, it would have to be templated on the three types and would require more effort on the reader's side to figure out where that function was. i.e. they'd have to scroll to find the function.

I tried to write these simple unit tests in a way that is as explicit as possible.

Reply:
Do you want me to move them? Or should I just leave it as is?  
Either way is fine with me, but this current way is already tested.

Reply:
Thanks Rich.  It might not be obvious from github, but the lambdas are in separate scopes -- each is defined within a `TEST()`, i.e. they are in separate functions.

(Side note: They also have different signatures, since the input parameters are templated on different dimensions)

Reply:
I didn't dig through the file, just looked at the PR changes in GitHub. 👍